Automattic/pocket-casts-android
# Update wear branch with latest from main

## Description
Pulling the latest changes from `main` into the `add/wear` branch.

## Testing Instructions
As long as CI can build this, no testing is really needed since the `add/wear` branch is still in an early development phase. At most, just give the watch app some light smoke testing to make sure there are no obvious crashes.
